packages/core/src/utilities/elementFromString.ts

```diff
@@ -14,9 +14,37 @@ const removeWhitespaces = (node: HTMLElement) => {
   return node
 }
 
+let policy = {
+  createHTML: (input: any) => input,
+  createScript: (input: any) => input,
+  createScriptURL: (input: any) => input,
+}
+
+try {
+  // @ts-ignore
+  // eslint-disable-next-line no-undef
+  policy = globalThis.trustedTypes.createPolicy('tiptap', {
+    createHTML: (input: any) => input,
+    createScript: (input: any) => input,
+    createScriptURL: (input: any) => input,
+  })
+} catch (error) {
+  // @ts-ignore
+  // eslint-disable-next-line no-undef
+  if (window.trustedTypes) {
+    // @ts-ignore
+    // eslint-disable-next-line no-undef
+    policy = window.trustedTypes.createPolicy('tiptap', {
+      createHTML: (input: any) => input,
+      createScript: (input: any) => input,
+      createScriptURL: (input: any) => input,
+    })
+  }
+}
+
 export function elementFromString(value: string): HTMLElement {
   // add a wrapper to preserve leading and trailing whitespace
-  const wrappedValue = `<body>${value}</body>`
+  const wrappedValue = policy.createHTML(`<body>${value}</body>`)
```

Comment:
Does prosemirror support this because I am not willing to use this if prosemirror does not even support this. 

A link to the source would be enough to convince me. Probably in prosemirror view since that touches the DOM